django-server/server/api/urls.py

```diff
@@ -3,7 +3,7 @@
 from . import views
 
 router = routers.DefaultRouter()
-router.register(r'cafeteria', views.CafeteriaViewSet)
+router.register(r'cafeterias', views.CafeteriaViewSet)
```

Comment:
Code added by accident? Please move to a separate PR.

Reply:
done in #89